rosdep/python.yaml

```
@@ -6079,6 +6069,8 @@ python3-rosdistro-modules:
   openembedded: [python3-rosdistro@meta-ros]
   rhel: ['python%{python3_pkgversion}-rosdistro']
   ubuntu: [python3-rosdistro-modules]
+python3-rosinstall-generator:
+  ubuntu: [python3-rospkg]
```

Without the explicit null entries this is still pointing to the wrong packages. `python3-rospkg` is not the same as `python3-rosinstall-generator`

```suggestion
  ubuntu: [python3-rosinstall-generator]
```

---

What a terrible mistake. Thank you for catching.